src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.block_stars {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Remove this. Only stars should have flex and the star wrappers must be under each other like all blocks by default.